src/pages/Home.vue

```diff
@@ -134,6 +134,6 @@ export default defineComponent({
 
 <style scoped>
 .invest-container {
-  @apply w-full max-w-screen-2xl;
+  @apply w-full max-w-screen-xl;
```

Comment:
How do you feel about reducing the max-width of the content sections on the home page? The tables feel too wide at the moment and I think this smaller layout keeps things compact and tidy.

Reply:
I think we should stick to the container class as I've done in the last commit, and mentioned above. That way the home and pool pages are consistent. I guess we could change the container dimensions across the board if needed. However, the tables don't feel that wide to me on the home page tbh and it gives us room to add more context/data later if needed.